lua/gruvbox_crisp/init.lua

```
@@ -0,0 +1,75 @@
+  vim.o.background = "dark"
+
+  local p = palette.dark
+
+  if o.terminal_colors then set_terminal(p) end
+
+  vim.g.colors_name = "gruvbox_crisp"
+
+  local specs = groups.get(p, o)
+  for name, spec in pairs(specs) do
+    if spec.link then
+      vim.api.nvim_set_hl(0, name, { link = spec.link, default = false })
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)